spec/helpers/application_helper_spec.rb

```diff
@@ -43,4 +43,32 @@
       end
     end
   end
+
+  describe '#localize_plural' do
+    let(:post) { create(:post) }
+
+    context 'with a popullated colection' do
+      context 'with one element' do
+        before { create(:comment, post: post) }
+
+        it 'returns a string with the quantity and the plural of the model' do
+          expect(localize_plural(post.comments, Comment)).to eq '1 comment'
+        end
+      end
+
+      context 'with more than one element' do
+        before { create_list(:comment, 2, post: post) }
+
+        it 'returns a string with the quantity and the plural of the model' do
+          expect(localize_plural(post.comments, Comment)).to eq '2 comments'
+        end
+      end
+    end
+
+    context 'without a popullated colection' do
```

Comment:
No problem! English spelling can be [difficult even for native speakers](https://aeon.co/essays/why-is-english-so-weirdly-different-from-other-languages).